java-test-samples/fargate-dynamodb-sync/spingboot-with-dynamodb/src/test/java/com/amazon/aws/sample/mapper/CustomerMapperTest.java

```diff
@@ -0,0 +1,31 @@
+package com.amazon.aws.sample.mapper;
+
+import com.amazon.aws.sample.model.Customer;
+import com.amazon.aws.sample.model.CustomerView;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class CustomerMapperTest {
+
+    private static final String customerId = "1";
```

Comment:
Reusing variables across tests is not a best practice

Reply:
Corrected as suggested, committed & pushed the changes.